x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/hunter/delete_detections_user.sh

```diff
@@ -0,0 +1,3 @@
+curl -v -H 'Content-Type: application/json' -H 'kbn-xsrf: 123'\
+ -u ${ELASTICSEARCH_USERNAME}:${ELASTICSEARCH_PASSWORD} \
+-XDELETE ${ELASTICSEARCH_URL}/_security/user/hunter
```

Comment:
These scripts share a lot between users/roles it looks like, mostly being just the referenced user/role string in the URL. Could this be extracted to an input arg and then we'd only have to maintain a single delete/get/post_role/post_user for all user/role jsons?

Reply:
My thinking behind having copies of the same scripts in each role directory is to allow anyone using these scripts to customize them in ways that could differ between roles. That way users could post two different users like a hunter and a T1 analyst but maybe not use the T1 analyst role, and try something a bit more / less restrictive for example. I think that level of granular customization despite repeating code between the roles would be helpful down the line.